tests/functional/sign_up.js

```diff
@@ -247,19 +225,11 @@ define([
           .type(PASSWORD)
         .end()
 
-        .findByCssSelector('#fxa-age-year')
-        .end()
-
-        .findById('fxa-' + CUTOFF_YEAR)
-          .click()
-        .end()
-
-        .findById('fxa-month-' + monthToSelect)
-          .click()
-        .end()
-
-        .findById('fxa-day-' + dateToSelect)
-          .click()
+        .findByCssSelector('#age')
+        // XXX: Bug in Selenium 2.47.1, if Firefox is out of focus it will just type 1 number,
+        // split the type commands for each character to avoid issues with the test runner
+        .type('1')
```

Comment:
When you do the `.type`, do you have to do a `.click` first so the element receives the focus? I could swear that's why we do stuff like https://github.com/mozilla/fxa-content-server/pull/3138/files#diff-addb444c51a96421c45bfd482b36edd8R224